### PR TITLE
Fix `selector-combinator-disallowed-list` reported ranges

### PR DESCRIPTION
**Pull request by ryo-manba**
> Which issue, if any, is this issue related to?

See https://github.com/stylelint/stylelint/issues/7904

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: lib/rules/selector-combinator-disallowed-list/__tests__/index.mjs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import rule from '../index.mjs';
+import { stripIndent } from 'common-tags';
 const { messages, ruleName } = rule;
```
Comment (romainmenke) [inline lib/rules/selector-combinator-disallowed-list/__tests__/index.mjs:3]:
```suggestion
import { stripIndent } from 'common-tags';

import rule from '../index.mjs';
const { messages, ruleName } = rule;
```

**Comment by ryo-manba** [comment]
Thanks for the review! Could you enable [automatic branch deletion](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches) after merges?

**Comment by romainmenke** [comment]
> Could you enable [automatic branch deletion](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches) after merges?

That is already enabled, but I don't think it affects forks, which makes sense :)

Screenshot 2024-10-15 at 14 15 02

**Comment by ryo-manba** [comment]
> That is already enabled, but I don't think it affects forks, which makes sense :)

Ah, right. Thanks for confirming.

**Review by romainmenke — APPROVED** [review]
LGTM 🎉

**Pull request merged**